src/app/main/component/events/components/create-edit-events/create-edit-events.component.ts

```diff
@@ -266,7 +266,7 @@ export class CreateEditEventsComponent implements OnInit, OnDestroy {
   }
   private checkUserSigned(): boolean {
     this.getUserId();
-    return Boolean(this.userId);
+    return this.userId ? true : false;
```

Comment:
What if userId is 0?

Reply:
yes, I should take it into account, thanks